wlauto/devices/android/juno/__init__.py

```diff
@@ -80,7 +80,8 @@ class Juno(BigLittleDevice):
                       'image_args': 'console=ttyAMA0,115200 '
                                     'earlyprintk=pl011,0x7ff80000 '
                                     'verbose debug init=/init '
-                                    'root=/dev/sda1 rw ip=dhcp rootwait',
+                                    'root=/dev/sda1 rw ip=dhcp rootwait'
```

Comment:
There needs to be a space after `rootwait`, otherwise the strings will get concatenated into `rootwaitvideo=DVI-D-1:1920x1080R@60`